runtime/browser/cdn/browser-cdn-loader.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+const Loader = require('../../loader.js');
+const particle = require("../../particle.js");
+const DomParticle = require("../../dom-particle.js");
+
+module.exports = class BrowserLoader extends Loader {
+  constructor(urlMap) {
```

Comment:
Do we still need this, What is expected to be mapped?

I was hoping for v0.1 that we would just reference things explicitly by URL. If we need to make things easier we can bundle up useful manifests of particles/recipes.